CHANGES/5924.misc

```diff
@@ -0,0 +1 @@
+Fix advisory conflict resolution to check current version first and enabled SyncMutatedUpdateRecordTestCase.
```

Comment:
I think the patch to enable the test was lost, because I don't believe it's enabled?

Reply:
Was not lost, there was intention to fix/enable it too but will need refactor of test itself (issue #6015 on pulp.plan.io) before just enable it.